lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -434,12 +440,52 @@ def refine_lines_gaussian(
         Fixed background parameter in photons per second.
         When supplied, the background is not estimated but fixed at this value.
     """
+    return refine_tracks_gaussian(
+        lines,
+        window,
+        refine_missing_frames,
+        overlap_strategy,
+        initial_sigma=initial_sigma,
+        fixed_background=fixed_background,
+    )
+
+
+def refine_tracks_gaussian(
+    tracks,
+    window,
+    refine_missing_frames,
+    overlap_strategy,
+    initial_sigma=None,
+    fixed_background=None,
+):
+    """Refine the tracks by gaussian peak MLE.
+
+    Parameters
+    ----------
+    tracks : List[pylake.KymoTrack] or pylake.KymoTrackGroup
+        Detected tracks on a kymograph.
+    window : int
+        Number of pixels on either side of the estimated track to include in the optimization data.
```

Comment:
To me, it seems strange that now we have one refinement method in pixels and one in physical units. I would make these consistent.

Reply:
Right. In that case it's actually good that it has a different name and the user's attention is drawn to it a bit. Fully agree with your assessment.